### DISCUSSION TITLE
Using user downloaded fonts in the app

### DISCUSSION THREAD

**Discussion by shajidhasan** [ionic-team/capacitor#4956] [Q&A]
Hello! I need to able to use user downloaded fonts in the app. My app will be a canvas based app that lets the user add texts in the canvas. That's why I need to find a way to use user downloaded ttf fonts in the app. Is it possible with capacitorjs?

**Comment by rashad** [comment]
I've tried and stubled upon some weird behaviour (NextJS, Tailwind, Capacitor 3)
- `.ttf` files seem to work better than `.otf` files
- no problem to make things work on the web
- setup on mobile is still a mystery (embed fonts in Xcode etc) 

If anyone has ever done this before, guidance would be much appreciated 🙏
